src/xGadgetron/cGadgetron/include/sirf/Gadgetron/TrajectoryPreparation.h

```diff
@@ -49,26 +49,108 @@ namespace sirf{
 * reconstruction. The ISMRMRD format has a 3D trajectory data field in their ISRMRMRD::Acquisition classe.
 * The interface provides set_trajectory() which populates this data field depending on the implementation.
 */
+template <uint16_t D>
```

Comment:
This should be `std::uint16_t` (also in other places). any specific reason we don't just use `unsigned`?